rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_compute_instance THAT !HAS Finding",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "MEDIUM"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-patch-vulnerabilities-azure",
+        "description": "M1048 - Application Isolation and Sandboxing - Make it difficult for adversaries to advance their operation through exploitation of undiscovered or unpatched vulnerabilities by using sandboxing in Azure. Other types of virtualization and application microsegmentation may also mitigate the impact of some types of exploitation. Risks of additional exploits and weaknesses in these systems may still exist.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND azure_vm THAT !HAS Finding",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "MEDIUM"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-minimize-service-account-permissions",
+        "description": "M1048 - Application Isolation and Sandboxing - Minimize permissions and access for service accounts to limit impact of exploitation.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_iam_service_account AS acct THAT ASSIGNED AS rel google_iam_binding AS binding RETURN acct.displayName, count(rel)",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "INFORMATIONAL"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-patch-management",
+        "description": "M1051 - Update Software - Update software regularly by employing patch management for internal enterprise endpoints and servers.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND HostAgent WITH automaticAppUpdates='OFF' OR automaticDownloadUpdates='OFF' OR automaticOsUpdates='OFF' OR automaticSecurityUpdates='OFF' OR automaticUpdates='OFF' OR autoSecurityPatchEnabled=false and autoSystemPatchEnabled=false",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-scan-for-vulnerable-services",
+        "description": "M1016 - Vulnerability Scanning - Regularly scan the internal network for available services to identify new and potentially vulnerable services.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND (Device|Host) THAT PROTECTS HostAgent WITH function=('av' OR 'anti-malware') AND active=true",
```

Comment:
flip the relationship direction, HostAgent -PROTECTS-> (Device|Host) is how the data model is defined

Reply:
corrected traversal direction